wemake_python_styleguide/violations/oop.py

```diff
@@ -574,3 +576,45 @@ def some_method(self):
         'Found incorrect `super()` call context: incorrect name access'
     )
     code = 613
+
+
+@final
+class UnpythonicGetterSetterViolation(ASTViolation):
+    """
+    Forbids to use getters and setters in objects.
+
+    Reasoning:
+        Python does not need this abstraction.
+
+    Solution:
+        Either use ``@property`` or make the
+        attribute public and change it directly.
+
+    Example::
+
+        # Correct:
+        class Example(object):
+            def __init__(self):
+                self._attribute = None
+
+        # Wrong:
+        class Example(object):
+            def __init__(self):
+                self.attribute = None
+
+            def set_attribute(self):
```

Comment:
`get` and `set` can have just `...` as implementation, because it does not matter.